pkg/core/mempool/mempool.go

```diff
@@ -65,37 +63,43 @@ type Mempool struct {
 	latestBlockTimestamp int64
 
 	eventBus *eventbus.EventBus
-	db       database.DB
+	// TODO: do we still need a DB in the mempool?
```

Comment:
Nope, as discussed, Rusk will check the DB for double-spends when verifying a mempool tx.

Reply:
Removed the comment